src/dosbox.cpp

```diff
@@ -622,6 +622,18 @@ void DOSBOX_Init()
 	        "320x200 or 640x400; whereas square-pixel modes, such as 640x480\n"
 	        "and 800x600, are displayed as-is.");
 
+	pstring = secprop->Add_string("integer_scaling", always, "off");
+	pstring->Set_help(
```

Comment:
(minor)
`integer_scaling` is added to the `[render]` section here, but looks like it's being read from the `[sdl]` section (in sdlmain).

Reply:
Yeah it was in `[sdl]` originally, then I asked @GranMinigun to move it to `[render]`.

Reply:
makes sense, thanks @johnnovak 

Once this conf match-up is ready I'll be able to check `horizontal` and `vertical`.

Reply:
I don't want to touch configuration ever again.

Should work now, but setting it at runtime requires manual triggering of viewport resize. Not quite sure yet how to do it at least remotely right.

Reply:
> I don't want to touch configuration ever again.
> 
> Should work now, but setting it at runtime requires manual triggering of viewport resize. Not quite sure yet how to do it at least remotely right.

You'll get used to it. If you think this is hard, imagine what I went through when I added the chorus, reverb, crossfeed & filters to the mixer AND the mixer commands to manipulate these all in realtime... 😅

Reply:
I don't think we have on-the-fly adjustable windowresolution or full screen toggling (via conf settings), either.  

Although they can be set at runtime, their values only kick in when the context is replaced (by changing output = ..)

Reply:
> You'll get used to it.

No.